android/build.gradle

```diff
@@ -16,6 +16,7 @@ buildscript {
 		classpath 'com.android.tools.build:gradle:7.0.4'
 		classpath 'com.google.gms:google-services:4.3.10'
 		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+		classpath 'org.codehaus.groovy:groovy-json:3.0.11'
```

Comment:
This looks like a third-party dependency (or even fork?), can we prevent that?

Reply:
this is because of https://github.com/tidev/titanium_mobile/pull/13473/files#diff-29c0a847a07a95fad8d2db898aca7db913cfa4a759621e5e80276111f59b0ac3R129 and was added by Studio when optimizing that line

Reply:
Okay, if Android Studio added it, it should be fine!